app/views/Import.js

```diff
@@ -86,15 +90,15 @@ const ImportScreen = props => {
             }
             style={styles.buttonTouchable}>
             <Typography style={styles.buttonText}>
-              {languages.t('label.import_takeout').toUpperCase()}
+              {languages.t('import.google.visit_button_text')}
             </Typography>
           </TouchableOpacity>
           <TouchableOpacity
             testID='google-takeout-import-btn'
             onPress={importPickFile}
             style={styles.buttonTouchable}>
             <Typography style={styles.buttonText}>
-              {languages.t('label.import_title').toUpperCase()}
+              {languages.t('import.title')}
```

Comment:
Same as above

Reply:
See above, it's not in the spec to have caps buttons